public/pages/DetectorsList/List/List.tsx

```diff
@@ -85,10 +99,10 @@ interface ListState {
 export const DetectorList = (props: ListProps) => {
   const dispatch = useDispatch();
   const allDetectors = useSelector((state: AppState) => state.ad.detectorList);
+  const allMonitors = useSelector((state: AppState) => state.alerting.monitors);
```

Comment:
Will this cause performance issue? Default monitor count is 1000, but user can change it. How about we limit how many detectors can be deleted in a batch call and get their related monitors when click start/stop button?

Reply:
Adding selection to the table limits the maximum number of selected detectors to be whatever maximum page size there is (in this case, 50). Selection can't span across multiple table pages. So, the batch limit here is 50. I've tested starting/stopping 50 detectors at once and it seems to not have major performance loss.